.travis.yml

```diff
@@ -10,6 +10,9 @@ jobs:
   include:
     - stage: e2e
       name: "e2e: docker with weave"
+      install:
```

Comment:
This will make travis slower unless installed gems are cached between builds.

Reply:
Also: what is the reason for repeating this in `.travis.yml` vs putting it into `travis.sh`?

Reply:
Reason is to replace the travis default `install: bundle install`, I suppose it could be set to null. 

I'm not certain `gem install` would use the cached `bundle install`ed gems anyway unless GEM_PATH is set to the bundle-dir.

Reply:
> 'm not certain gem install would use the cached bundle installed gems anyway unless GEM_PATH is set to the bundle-dir.

 Yeah, I meant that probably we should cache gems folder for faster gem install.